romana-install/roles/romana/install-master/files/build_romana_core

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+export GOPATH=/var/tmp/gopath
+export PATH=/usr/local/go/bin:$PATH
+
+go get -d github.com/romana/core/...
```

Comment:
That part happens here:
https://github.com/romana/romana/blob/build-core-from-source/romana-install/roles/romana/install-master/tasks/romana_from_github.yml#L15
Should support branches, tags, commit hashes etc.